README.md

```diff
@@ -20,9 +20,8 @@ Collection of Maven plug-ins and archetypes for managing WebSphere Application S
 Use Maven 3.x to build the Liberty plug-ins and archetypes.
 
 * `mvn install` : builds the plug-in and the archetypes.
-* `mvn install -Poffline-its -DwlpInstallDir=<liberty_install_directory>` : builds the plug-in and the archetypes and runs the integration tests by providing an existing installation.
-* `mvn install -Ponline-its -DwlpVersion=<liberty_version> -DwlpLicense=<liberty_license_code>` : builds the plug-in and archetypes and runs the integration tests by downloading a new server.
-  * Liberty versions and their respective link to the license code can be found in the [index.yml](http://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml) file. You can obtain the license code by reading the current license and looking for the D/N: <license code> line.
+* `mvn install -Poffline-its -DlibertyInstallDir=<liberty_install_directory>` : builds the plug-in and the archetypes and runs the integration tests by providing an existing installation.
+* `mvn install -Ponline-its -Druntime=<ol|wlp> -DruntimeVersion=<runtime_version>` : builds the plug-in and archetypes and runs the integration tests by downloading a new server. Set runtime to `ol` to run tests using the Open Liberty runtime, or `wlp` to run tests using the WLP Java EE 7 runtime.
```

Comment:
So how will they specify a license for wlp now?

Reply:
That parameter actually wasn't even being used. If you do a search in the repository for `wlpLicense` there is only one result. It is being passed as a parameter to our tests but never being used.